_content/doc/tutorial/getting-started.html

```diff
@@ -9,7 +9,7 @@
 </p>
 
 <ul>
-  <li>Install Go (if you haven't already).</li>
```

Comment:
@willpoint kindly help deploy this version as I've made the changes requested. Thank you!